File: FbxCppTests/DeformerTest.cpp

#include "Tests.h"

using namespace std;

void Deformer_Create_HasNamespacePrefix()
{
    // given:
    FbxManager* manager = FbxManager::Create();
    FbxSkin* obj = FbxSkin::Create(manager, "asdf");

    // then:
    AssertEqual("Deformer::", obj->GetNameSpacePrefix());;
}

void DeformerTest::RegisterTestCases()
{
    AddTestCase(Deformer_Create_HasNamespacePrefix);
}

File: FbxSharpTests/DeformerTest.cs
using System;
using NUnit.Framework;
using FbxSharp;

namespace FbxSharpTests
{
    [TestFixture]
    public class DeformerTest : TestBase
    {
        [Test]
        public void Deformer_Create_HasNamespacePrefix()
        {
            // given:
            var obj = new FbxSkin("asdf");

            // then:
            Assert.AreEqual("Deformer::", obj.GetNameSpacePrefix());;
        }
    }
}
